core/services/ardupilot_manager/main.py

```diff
@@ -52,6 +52,17 @@
     raise RuntimeError("ArduPilot manager needs to run with root privilege.")
 
 
+def target_board(board_name: Optional[str]) -> FlightController:
+    if board_name is not None:
```

Comment:
It would be nice to have comments explaining how this function operates and the reason behind board_name to be used when the board is in a bootloader state.

Reply:
Will add some comments.